plugins/woocommerce/src/Internal/CLI/Migrator/Platforms/Shopify/ShopifyMapper.php

```diff
@@ -487,8 +518,8 @@ private function map_variable_product_data( object $shopify_product, bool $is_va
 		if ( $is_variable && property_exists( $shopify_product, 'options' ) && ! empty( $shopify_product->options ) ) {
 			foreach ( $shopify_product->options as $option ) {
 				$variable_data['attributes'][] = array(
-					'name'         => $option->name,
-					'options'      => $option->values,
+					'name'         => sanitize_text_field( $option->name ),
+					'options'      => array_map( 'sanitize_text_field', $option->values ),
```

Comment:
Nit. The `wc_clean()` function has the same logic :) We could replace it here for consistency.